core/unit_test/TestTeam.hpp

```diff
@@ -1642,9 +1642,12 @@ struct TestScratchAlignment {
           // and since scratch_ptr3 is then already aligned it difference
```

Comment:
Comment is not very understandable; by, "...and since scratch_ptr3 is already aligned, its difference should match that," do you mean that the scratch_ptr3 shmem allocation, by virtue of being a multiple of 32, will be aligned?

Reply:
I didn't touch that.

Reply:
OK, but can we fix it up to make sense (assuming we understand the original intent)?  Just aiming for improvement, while we're in the neighborhood

Reply:
See https://github.com/kokkos/kokkos/pull/5800.